### PR TITLE
Day nineteen

### PR DESCRIPTION
**Pull request by Qairkhan**
Добавлен функционал авторизации.

### CODE REVIEW

File: src/components/Header/HeaderContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import axios from "axios";
+
+import Header from "./Header";
+import { connect } from "react-redux";
```
Comment (Dmitry-White) [inline src/components/Header/HeaderContainer.jsx:3]: Consider fixing the import grouping here.

File: src/redux/actionCreators.js [hunk 2]
```diff
@@ -50,6 +50,11 @@ const setUserProfile = (payload) => ({
   payload,
 });
 
+const setUserData = (userId, email, login) => ({
+  type: ACTION_TYPES.SET_USER_DATA,
+  data: { userId, email, login },
```
Comment (Dmitry-White) [inline src/redux/actionCreators.js]: And again, don't forget that action object should contain "type" and "payload" keys.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/components/Header/HeaderContainer.jsx (hunk 1 above)
Comment (Qairkhan) [reply src/components/Header/HeaderContainer.jsx:3]: done

File: src/redux/actionCreators.js (hunk 2 above)
Comment (Qairkhan) [reply src/redux/actionCreators.js]: done

**Review by Dmitry-White — APPROVED** [review]

**Pull request merged**